website/components/mdx/_publish-python.mdx

```diff
@@ -0,0 +1,19 @@
+
+2. Your plugin code needs to specify the plugin name, version, team name, and plugin type in the `__init__` function. See example from our [plugin template](https://github.com/cloudquery/python-plugin-template) below:
```

Comment:
This seems to start with `2.` instead of `1.`, is there a step missing, or just the numbering is wrong? It'll probably show up correctly in rendered markdown but either way would be good to fix it